asset-transfer-secured-agreement/README.md

```diff
@@ -0,0 +1,342 @@
+# assets private asset transfer scenario
```

Comment:
This is an awkward title. The phrase assets transfer is also awkward. I would rename the title, secure private asset transfer scenario, and call it the secure asset transfer smart contract.

Reply:
🛠️